docs/metasploit-framework.wiki/Nightly-Installers.md

```diff
@@ -30,6 +30,33 @@ Download the [latest Windows installer](https://windows.metasploit.com/metasploi
 
 If you downloaded Metasploit from us, there is no cause for alarm.  We pride ourselves on offering the ability for our customers and followers to have the same toolset that the hackers have so that they can test systems more accurately.  Because these (and the other exploits and tools in Metasploit) are identical or very similar to existing malicious toolsets, they can be used for nefarious purposes, and they are often flagged and automatically removed by antivirus programs, just like the malware they mimic.
 
+## Windows Installation (Silent)
+
+The PowerShell below will download and install the framework, and is suitable for automated Windows deployments. 
+```
+[CmdletBinding()]
+Param(
+    $DownloadURL = "https://windows.metasploit.com/metasploitframework-latest.msi",
+    $DownloadLocation = "$env:APPDATA/Metasploit",
+    $InstallLocation = "C:\Tools",
```

Comment:
Let's double check that the installer will work with this target location. See https://github.com/rapid7/metasploit-omnibus/issues/144

Reply:
I checked and the target installation path is correct (`C:\Tools`).